pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
     "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-    "aioice>=0.9.0,<1.0.0",
+    "aioice>=0.10.1,<1.0.0",
```

Comment:
necessary since we depend on 0.10.x to pass username and password to the gatherer.